### ISSUE TITLE
Wallpaper uploading on Samsung Galaxy S3 inconsistent

### ISSUE THREAD

**Issue by geoffbrain** [CommandFusion/iViewer-For-Android#76]
Release 102 has been very good I have not found any serious issues.  One issue I have found is the loading of Wallpapers on Samsung Galaxy S3 inconsistent.  Once the GUI is loaded all works including those wallpapers that do load.  

However during the upload process to ensure every wallpaper is loaded I have to go to each page that has its own unique wallpaper, this seems to force that wall paper to be uploaded if it has not been already uploaded.  If I do not do this of the 18 or so wallpapers I have a few do not get uploaded once the guiDesigner is closed.

The same applies to some subpages.

The results are not the same for each upload but appear random between each upload.

I hope this makes sense.

**Comment by geoffbrain** [comment]
I also have been using a Bauhn AMid-972S Android 4 tablet and it exhibits the same problem.  To ensure I have all wallpapers and dialogs (subpages) backgrounds I have to display them while uploading. In fact I think all images (including buttons) if not displayed don't seem to be uploaded.  I do not see this with button normally as I only use 10 or so buttons and they are all bar one on the main subpage. If I don't do this not all are uploaded.  I cannot seem to see any common sequence each time I do this.

**Comment by fpillet** [comment]
Would you mind having a go to the latest build and let me know if this issue still occurs for you? thanks!

**Comment by geoffbrain** [comment]
Sorry but I have forgotten where to download the latest beta from - can you point me there.  Ta

**Comment by fpillet** [comment]
Document with instructions and download links: http://goo.gl/4jRTr

**Comment by fpillet** [comment]
Any update on this issue with the latest builds? Is this still happening?

**Comment by geoffbrain** [comment]
Sorry I have been away, will be able to test tonight.  Will post a response.

**Comment by geoffbrain** [comment]
Ok one good think no crashes.

But when I force the reload of a GUI it only loads the images that are on the current page  and some others but not all and any displayed subpages.  If I select a new page or subpage these images are then loaded and the green activity light on GUIDesigner blinks green.

If I swap to new pages or subpages and the images have not been downloaded they green activity light lights up on the GUI Designer and the images are downloaded.  As best I can tell all images are not downloaded in one bundle as they should be. I'll experiment a little more tomorrow.

I hope mu description this makes sense,

**Comment by fpillet** [comment]
I'd be interested in trying out your GUI, could you email it to me? florent at commandfusion. Thanks!

**Comment by geoffbrain** [comment]
Florent,

To which email address do you want the GUI sent?  I assume all images as
well?

Regards,

Geoff Brain
s

 Phone: 61-7-3032-6030 | Mobile: 61-412-207-560 |  
 Home: 61-7-5526-8857  
 E-mail: brain@au1.ibm.com

**Comment by geoffbrain** [comment]
OK I had another play.

this time I let the GUI reload from the GUIDesigner upload service for good 10 minutes before using it.  It is hard to see what is happening but it seems not all images are uploaded.  It seems that any that are not on the current screen displayed are only uploaded when required.  I tried stopping the upload services and any images not uploaded were not displayed.  If I switched the upload services on again they were uploaded and what was strange is the Connected Device ID filed was not populated with the license code but the activity light would still flash green when the image was being downloaded.  So all up quite strange and hard to explain.

I have sent the files to you.
